.vitepress/config.mts

```diff
@@ -8,43 +8,73 @@ import ViteUnoCSS from 'unocss/vite'
 import { mermaid } from './md-mermaid'
 import { katex } from '@mdit/plugin-katex'
 
-function getGuideSidebar(): DefaultTheme.SidebarItem[] {
+function nav(): DefaultTheme.NavItem[] {
   return [
     {
-      text: 'Getting started',
+      text: 'Get Started',
+      link: '/get-started/index',
+      activeMatch: '/get-started/',
+    },
+    {
+      text: 'Developer Guide',
+      link: '/guide/introduction',
+      activeMatch: '/guide/',
+    },
+    {
+      text: 'API Reference',
+      link: '/api/index',
+      activeMatch: '/api/',
+    },
+    {
+      text: 'Troubleshooting',
+      link: '/guide/troubleshooting/overview',
+      activeMatch: '/guide/troubleshooting/',
+    },
```

Comment:
I suggest [nesting](https://vitepress.dev/reference/default-theme-nav#navigation-links) it into __Developer Guide__ item:

- Developer Guide
  - Introduction
  - Troubleshooting